app/models/form/question.rb

```diff
@@ -70,9 +70,9 @@ def has_inferred_check_answers_value?(case_log)
 
   def update_answer_link_name(case_log)
     if type == "checkbox"
-      answer_options.keys.any? { |key| case_log[key] == "Yes" } ? "Change" : "Answer"
+      answer_options.keys.any? { |key| case_log[key] == "Yes" } ? "Change<span class=\"govuk-visually-hidden\"> #{check_answer_label.to_s.downcase}</span>".html_safe : "Answer<span class=\"govuk-visually-hidden\"> #{check_answer_label.to_s.downcase}</span>".html_safe
     else
-      case_log[id].blank? ? "Answer" : "Change"
+      case_log[id].blank? ? "Answer<span class=\"govuk-visually-hidden\"> #{check_answer_label.to_s.downcase}</span>".html_safe : "Change<span class=\"govuk-visually-hidden\"> #{check_answer_label.to_s.downcase}</span>".html_safe
```

Comment:
Don’t know if this can be made dryer?

Reply:
I have an idea, will test and update this thread if it works

Reply:
@paulrobertlloyd I've managed to reduce the repetition and it still passes all the tests 😄

Reply:
It's in commit d5ed74c0676d6ad17b576ebf7a9725ba6a741fa1 if you're curious